geniza/entities/templates/entities/person_detail.html

```diff
@@ -0,0 +1,70 @@
+{% extends 'base.html' %}
+{% load static i18n render_bundle_csp fullurl %}
+
+{% block meta_title %}{{ page_title }}{% endblock meta_title %}
+{% block meta_description %}{{ page_description }}{% endblock meta_description %}
+{% block extrameta %}
+    {% spaceless %}
+        {% for lang_code in PUBLIC_SITE_LANGUAGES %}
+            {% language lang_code %}
+                <link rel="alternate" hreflang="{{ lang_code }}" href="{% fullurl "entities:person" person.pk %}" />
```

Comment:
not strictly related but question: I saw an alert in google search console that we have alternate links without a canonical; do you know if we're declaring a canonical link anywhere in the header or how we would determine that one? do we have to declare one language as canonical?

Reply:
Good question! Looks like on document pages, we have `document.permalink` as the canonical page, and that property is generated by specifically stripping out the language from the URL. Not sure if that's how it's meant to be, since that will cause an immediate redirect to one with a language (presumably to whatever the crawler's language is), and therefore isn't necessarily stable in content. It looks like Google in particular does not like 3xx redirects on canonical pages, and prefers the result of the redirect as the canonical URL. So it may make sense to choose a language (presumably English) as canonical instead, on both documents and people.

Reply:
oh wow, is that what Google has been complaining about with the non-canonical?! good catch. I wonder if we can just add the `rel=nofollow` attribute to tell bots not to crawl it?  I do think that for SEO purposes our English pages should probably be the canonical version...

My intention was a language-agnostic permalink for citation purposes. I don't like the idea of privileging one language as canonical when citing the resource; if you're citing a PGP document, are you citing the English version of the description of that document? (maybe, sometimes) Are you citing the document itself? If you and someone else both reference a PGP document but looked at different versions of the localized site, should they be different urls or not? I can do some looking around and asking to see if there are any best practices for multilingual digital scholarship. Would be interesting to hear what Geniza Lab team members think about this.  (I'm thinking of the permalink more as a URI, if that make sense.)

I'm not sure if the same considerations apply to people as they do to documents.